installer/configurators/jenkins.py

```diff
@@ -0,0 +1,78 @@
+import subprocess
+import os
+import sys
+from pathlib import Path
+from installer.configurators.common import get_installer_root, get_tfvars_file, replace_tfvars
+
+jenkins_pem = get_installer_root() + "/jenkinskey.pem"
+
+
+def check_jenkins_pem():
+    """
+        Check if the user has provided jenkinskey.pem
+    """
+    print(
+        " Please make sure that you have the ssh login user names of jenkins and bitbucket servers."
+    )
+    # Check if user dropped the jenkins key into the installer root (scenario 1 only)
+    if not Path.is_file(jenkins_pem):
```

Comment:
Got error on this line, seems it should be "if not Path(jenkins_pem).is_file()"

Reply:
Correct, fixed.